main/main.gd

```diff
@@ -82,6 +82,16 @@ func create_players(num):
 		player.sprite.texture = sprites[i]
 		player.sprite.visibility_layer = 9
 		
+		var regexPattern = "\\b(?!res\\b)\\w+\\b"
+		var regex = RegEx.new()
+		regex.compile(regexPattern)
+		var matches = regex.search_all(sprites[i].resource_path)
+		
+		player.player_name = matches[1].get_string()
+		
+	var duel = preload("res://duel/duell_movement/overtaking_duell.tscn").instantiate()
```

Comment:
Whats the purpose of this?

Reply:
Maybe testing? Not sure anymore, but I think it can be removed.